dags/hivemind_etl_helpers/ingestion_pipeline.py

```diff
@@ -0,0 +1,68 @@
+from hivemind_etl_helpers.src.utils.credentials import load_redis_credentials
+from llama_index.core import MockEmbedding
+from llama_index.core.ingestion import (DocstoreStrategy, IngestionCache,
+                                        IngestionPipeline)
+from llama_index.core.node_parser import SemanticSplitterNodeParser
+from llama_index.storage.docstore.postgres import PostgresDocumentStore
+from llama_index.storage.kvstore.redis import RedisKVStore as RedisCache
+from llama_index.vector_stores.postgres import PGVectorStore
+from tc_hivemind_backend.db.credentials import load_postgres_credentials
+from tc_hivemind_backend.db.utils.model_hyperparams import (
+    load_model_hyperparams
+)
+from tc_hivemind_backend.embeddings.cohere import CohereEmbedding
+
+from dags.hivemind_etl_helpers.src.db.gdrive.db_utils import setup_db
+
+
+class CustomIngestionPipeline:
+    def __init__(self, community_id: str, table_name: str,
+                 testing: bool = False):
+        redis_credentials = load_redis_credentials()
+        postgres_credentials = load_postgres_credentials()
+        self.redis_host = redis_credentials["host"]
+        self.redis_port = redis_credentials["port"]
+
+        self.table_name = table_name
+        self.dbname = f"community_{community_id}"
+        self.db_config = {
```

Comment:
Please consider using the db_config directly rather than creating the `db_config` variable.